test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -71,3 +72,30 @@
         (is (nil? (get catalog "code_id")))
         (is (logged? #"Non-zero exit code returned while running" :error))
         (is (logged? #"Executed an external process which logged to STDERR: production" :warn)))))))
+
+(deftest ^:integration static-file-content-endpoint-test
+  (logging/with-test-logging
+    (testing "the /static_file_content endpoint successfully streams file content"
+      (bootstrap/with-puppetserver-running
+        app
+        {:jruby-puppet
+         {:max-active-instances num-jrubies}
+         :versioned-code
+         {:code-content-command (script-path "echo")}}
+        (let [response (http-client/get "https://localhost:8140/puppet/v3/static_file_content/foo/bar/?code-id=foobar&environment=test"
```

Comment:
I based the URL structure off of [PUP-5114](https://tickets.puppetlabs.com/browse/PUP-5114). Looks like that should definitely be `code_id` rather than `code-id`. @joshcooper Is the URL structure in PUP-5114 still what you all are looking for?